software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcessWinRmDriver.java

```diff
@@ -80,9 +80,15 @@ public void launch() {
 
     @Override
     public boolean isRunning() {
-        int exitCode = executeCommandInTask(
-                getEntity().getConfig(VanillaWindowsProcess.CHECK_RUNNING_COMMAND),
-                getEntity().getConfig(VanillaWindowsProcess.CHECK_RUNNING_POWERSHELL_COMMAND), "is-running-command");
+        int exitCode = 1;
+        try {
+            exitCode = executeCommandInTask(
+                    getEntity().getConfig(VanillaWindowsProcess.CHECK_RUNNING_COMMAND),
+                    getEntity().getConfig(VanillaWindowsProcess.CHECK_RUNNING_POWERSHELL_COMMAND), "is-running-command");
+        } catch (javax.xml.ws.WebServiceException/*Wraps Soap exceptions*/|org.apache.cxf.interceptor.Fault/*Wraps IO exceptions*/ e) {
+            LOG.warn(getEntity() + " isRunning check failed. Executing WinRM command failed.", e);
```

Comment:
It's worth including a comment with the reason here (as you did in the description of the PR). For example:

    // If machine is restarting, then will get WinRM IOExceptions - don't propagate such exceptions